spec/features/authenticated_user/authenticated_user_can_login_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.